README.md

```diff
@@ -63,3 +63,27 @@ module.exports = webpackConfig;
 **Compiled with errors**
 
 ![image](https://user-images.githubusercontent.com/1018196/35707809-14e12926-0760-11e8-9f67-af1eff6048ae.png)
+
+## Customizing
+
+See `index.js` for examples.
+
+- `window.__webpackEventColors__` [Object] Override default colors.
```

Comment:
This is cool, I do want to give some thought to how we should handle progress events. Webpack-dev-server doesn't currently route that to the handler, not sure why but it should I'll send out a PR to them sometime.

Rather than a simple border color change, we'd ideally want to inject position fixed dom element that grows in width proportional to the progress % of webpack dev server.

Reply:
I don't think a fixed element would be the best idea, it can cause so much bugs in any browser.

Reply:
What sort of bugs do you mean? ‘Fixed’ has been fixed in most newer browsers. I tried in safari, chrome, Firefox and it seems fine. 

Older browsers will fall back to absolute and that should work fine too. 

https://caniuse.com/#search=fixed

Reply:
I wasn't talking about browser support. The browser's composition mechanism can become bugged simply by having a `position: fixed` element on the page. There are many topics can be found about this in Google (scrolling-, render issues). In most cases `backface-visibility: hidden` and `translate3d()` are solving these issues but using `translate3d()` is usually another performance hit for slower machines, especially for low end phones. Since we're developing for such devices I'll probably override fixed to prevent such headaches.

Reply:
Fair. Added fallback to absolute